some improvements for German version

ghostgroove opened the pull request:
- corrected some typos and missing/wrong translations
- changed some wording for better comprehensibility
- replaced the English h for hours with the German Std. for Stunde(n)
- replaced "in Nähe" with "in der Nähe" due to the much higher general linguistic usage

brunobord commented:
maybe @mattschaffrath could review this, to make sure he agrees with these improvements?

mattschaffrath commented:
Nice to have someone else contributing. Overall the changes by @ghostgroove are improvements, catching some oversights on my part. But I disagree on the "in der Nähe" issue and some other minor choices. I suggest @ghostgroove and me discuss these issues, hopefully arriving at some common vision for a German translation. I would like to work together with @ghostgroove in the forked repo on a version we can both agree on.

ghostgroove commented:
Sounds great @mattschaffrath!
Since I'm using GitHub for the first time, where do we discuss? :blush:

brunobord commented:
>  Since I'm using GitHub for the first time, where do we discuss? :blush:

Why not here on this pull-request? Pull-requests are often used as a *medium* for discussing the contribution. As a plus, people would have access to the history and understand why you've decided to chose one term over another.

You can even discuss this in German, since it's a German argument.

ghostgroove commented:
* In Nähe vs. In der Nähe
Im englischen Original wird z.B. bei Moments/Minutes darauf geachtet einen normalen Sprachfluss der Regelbegriffe zu ermöglichen. Ich würde im normalen Sprachgebrauch eher "In der Nähe" benutzen, statt "In Nähe" und würde deswegen diese Änderung vorschlagen.

* h vs. Std.
Ich vermute hier sind wir uns einig?

* 1-händig/2-händig vs. einhändig/zweihändig
Falls wir uns nicht einig sind: Die erste Form steht nicht im Duden, weswegen ich eine Änderung vorschlagen würde.

* Bogen und Einhandwaffe in der Ausrüstungsliste
Hatte ich sogar vergessen zu erwähnen, erleichtert aber die Nutzung der Tabelle (man muss nicht erst in die FAQ schauen) und David hat ja selbst geschrieben das er die Einträge schlicht vergessen hat.

Hast du sonst noch Punkte/Fragen @mattschaffrath?

mattschaffrath commented:
Generell hatte ich bei der Übersetzung mehrere Leitgedanken, mit denen ich gearbeitet habe:
1. Möglichst Alltagssprache und allgemein gebräuchliche Worte verwenden (wobei ich da ein rein subjektives "Sprachgefühl" als Maß nehme und mir klar ist, dass je nach Sprachregion und Prägung teilweise gewaltige Unterschiede in dem bestehen können "was man so sagen kann").
2. Dabei bestimmte fixe Spieltermini durch konsistente Verwendung prägen.
3. Bestehende "Rollenspiel-Fachbegriffe" und vorherige D&D-Übersetzungen größtenteils ignorieren und nur dann verwenden, wenn sie allgemeinsprachlich vertretbar/die bessere Option sind.
4. Halbwegs geschlechtsneutrale Sprache verwenden. Ich kann mich bisher nicht mit den ganzen Vorschlägen mit Binnenzeichen für Person*_Innen anfreunden und bevorzuge die "Plural-Lösung", mit der im anglophonen Bereich oft bestimmte Geschlechter vermieden werden. Wenn es irgendwie auch nur ansatzweise vertretbar ist, vermeide ich Formulierungen wie "ein Charakter tut... er kann..." zu Gunsten von "Charaktere tun... sie können".
5. Da das hier im Rahmen eines mehrsprachigen Übersetzungsprojekts geschieht, will ich mich in Gliederung und Struktur stark in die englische Vorlage halten, in der Hoffnung, damit konform mit den anderen Sprachen (die ich alle nicht spreche) zu bleiben.

Das vorweg, konkret zu den Änderungen:

`the-black-hack.md`:

* **h vs. Std.:** Dein Vorschlag ist besser, wäre ich selbst gern drauf gekommen.
* **1-händig vs. einhändig:** Anscheinend machen die anderen Übersetzungen es auch so, also nichts dagegen, die gängige Schreibweise zu verwenden.
* **Bogen und Einhandwaffen in Ausrüstungsliste:** Da hatte ich drüber nachgedacht, aber mich dann dagegen entschieden. Grund: Ich versuche nicht, einen eigenen "German Hack" zu produzieren (dann würde ich bei Struktur und Formulierung vieles ändern), sondern eine Übersetzung des englischen Grundregelwerks zu bieten, wie es z.B. auch als PDF oder im Druck zu kaufen ist. Wenn der Autor des Grundspiels es vorzieht, die Druckversion final zu halten und das lieber in seinem Ergänzungs-PDF korrigiert als die Tabelle im Grundspiel zu aktualisieren, und hier alle anderen Übersetzungen es ebenfalls so halten, dann will ich eigentlich nicht mit der deutschen Übersetzung aus dem Schema ausbrechen und es anders machen, als der Urheber es anscheinend wünscht und vorsieht.
* **in (der) Nähe:** Eben weil das ein Alltags- und kein Fachbegriff ist, ist mir wichtig, dass er in einer fixen Form erkennbar und benutzbar bleibt. Mein Problem ist, dass die Entfernung sowohl als Ort (wo? - in *der* Nähe) als auch als Ziel (wohin? - in *die* Nähe) vorkommen kann - z.B. wenn sich ein **entfernter** Gegner **in die Nähe** eines Charakters bewegt. Das lässt sich mit Konstrukten wie "Ort in der Nähe" auch umgehen, aber es ist nicht damit getan, "in Nähe" einfach überall mit "in der Nähe" zu ersetzen.  
Generell bin ich aber mit den Entfernungsbezeichnungen nie ganz zufrieden gewesen, daher bin ich hier besonders für Verbesserungen offen.
* **Zeile 27:** auf das "und" kann man verzichten, aber ich meine, das Komma müsste zumindest bleiben
* **Zeile 41:** ich bevorzuge manchmal Ziffern gegenüber ausgeschriebenen Zahlen, wenn ich das Gefühl habe, dass die Anzahl/Menge eine Rolle spielt, z.B. bei Kochrezepten (2 Eier, 200g Mehl, 3 EL Öl etc.), hier kann man sich drüber streiten, ob das so ein Fall sein müsste.
* **Zeile 69:** Großer Schild sollte (nach Vorlage) mehr kosten und mehr Rüstungspunkte haben.
* **Zeile 110:** Das ist so ein Punkt, wo ich wie in 4) beschrieben den Plural gewählt habe. Ich finde, bei deiner Änderung ist es nicht 100% eindeutig, ob sich die Charaktere kollektiv eine Handlung/Bewegung teilen, wenn die Spieler an der Reihe sind, oder ob jeder Charakter einzeln handeln/sich bewegen darf. Vor allem, weil ich mit dem System mehrere "Funnel" geleitet habe, wo jeder Spieler drei oder mehr Charaktere führt.
* **Zeile 166 ff:** Ja, Ziffer macht es lesbarer.
* **Zeile 189:** ich denke, das "dass" kann nicht einfach wegfallen.
* **Zeile 213:** entweder "Waffen ... aufgelistet stehen" oder "Waffe ... aufgelistet steht", "Waffe ... aufgelistet stehen" geht nicht.
* **Zeile 238:** Im Satz vorher waren es auch mehrere Kleriker, warum plötzlich nur noch der eine?
* **Zeile 321:** Alles mit "zaubern" würde ich möglichst gern aus dem Klerikerbereich heraushalten (was mir offenbar auch nicht 100% geglückt ist). Denn wenn es eine "Zauberer"-Klasse gibt, die aus ihrem Zauberbuch Zauberer-Zaubersprüche zaubert, dann hätte ich lieber einen Kleriker, der aus seinem Heiligen Buch Göttliche Anrufungen betet, als ein Kleriker, der aus seinem Zauberbuch Kleriker-Zaubersprüche zaubert. Einerseits will ich die beiden Spruchklassen damit inhaltlich ein bisschen abgrenzen, andererseits die Begriffe im Spiel eindeutiger (und vielleicht weniger kalauerhaft) machen - "Ihr findet ein Zauberbuch." - "Ein Kleriker-Zauberbuch?" - "Nein, ein Zauberer-Zauberbuch." Auch hier bin ich für Verbesserungen offen.
* **Zeile 416:** "in (der) Nähe" gehört zu den Zielen und sollte dabei stehen. Zumindest da, wo ich herkomme, klingen Sätze wie "lässt 1W4 Pizzen backen im Ofen" oder "lässt 1W4 Kinder spielen im Schwimmbecken" komisch.
* **Zeile 417:** ist schon in meiner Version Quatsch, der mir durchgegangen ist - sollte heißen "umhüllt alles in Nähe eines einzelnen Ziels"
* **Zeile 418:** "Sprechen"/"Reden" klappt aus meiner Sicht gleichwertig, sollte aber dann konsequent bei Tieren, Toten und Monstern dasselbe Wort sein.
* **Zeile 433:** Deine Änderung lese ich wie "Entfernt Gift .... wirkungslos". Aber ja, schwieriger Satz, da sich der Zauber immer auf ein Ziel in (der) Nähe bezieht und bei ihm bereits wirkendes Gift entfernt und ihm innerhalb der Spruchdauer neu verabreichtes Gift wirkungslos macht. Sollte man vielleicht im Deutschen komplett umkrempeln.
* ** Zeile 543:** Ich bin für den "Doppelgänger", nicht den "Dopplegänger", das wäre ein abenteuerlicher sprachlicher Reimport.

Mir geht es mit den Punkten darum, meinen Standpunkt zu erklären, nicht unbedingt durchzusetzen. Dass ich den Text übersetzt habe, heißt vor allem, dass ich Zeit und Lust dazu hatte, nicht, dass ich durch irgendeine Qualifikation oder Autorität die beste Wahl dafür wäre. Daher bin ich an Meinungen und Verbesserungsvorschlägen sehr interessiert.

@ghostgroove Du solltest außerdem deinen Namen irgendwie in die Dokumente und die Metadatei einfügen (Lektorat, zusätzliche Übersetzung, Überarbeitung, nenn es wie du willst), wenn du schon mitarbeitest. Wenn dein Beitrag akzeptiert wird, heißt das, dass in der aktuellen Version Arbeit von dir drinsteckt und dann sollte da auch dein Name stehen. Wenn dein Beitrag aus irgendeinem Grund nicht akzeptiert würde, würden deine Änderungen sowieso keinen Eingang in das Projekt finden und dann wäre es auch egal, wenn du deinen Namen reingeschrieben hast. Also kein Grund, den rauszulassen! 😉

ghostgroove commented:
Deine grundsätzlichen Überlegungen wirken sehr durchdacht und darauf könnte ich mich ebenfalls festlegen.

Ein Hinweis aber doch zu 3.: Sicher ist das eine valide Möglichkeit, aber es stellt sich für mich die Frage, ob die gängige Übersetzung (Konstitution) nicht sinnvoller bleibt, da die Spieler ja damit rechnen. Wenn man sich bewusst dagegen entscheidet (womit ich wie gesagt kein Problem habe), wäre vielleicht auch Körperkraft statt Stärke eine Idee.

Frage zwischendrin: Kann ich jetzt meinen Patch editieren und der Pull wird automatisch angepasst (solange er offen ist)?

` the-black-hack.md:`
* **Bogen und Einhandwaffen in Ausrüstungsliste:** Okay, da hast du Recht.
* **in (der) Nähe:** Ich habe beim Editieren ein nahebei gesehen, wäre das nicht eine Option?
* **Zeile 27:** Hm, ja, mit dem Komma könntest du Recht haben.
* **Zeile 41:** Geht mir meistens auch so, aber gerade bei 1 empfand ich hier die geschriebene Version passender.
* **Zeile 69:** Ja, ist doch auch so. Oder spinnt meine Ansicht wo folgendes steht: > | Kleiner/Großer Schild        | 50/100 |       -       |     2/4 Rüstungspunkte     |
* **Zeile 110:** Ich hatte eigentlich nur aus Charakte Charaktere gemacht. Sieht in meiner Ansicht auch immer noch so aus. Ist das in deiner anders?
* **Zeile 189:** Da ist bei mir keine Änderung ...
* **Zeile 213:** Bei mir steht Waffen und stehen ...

Vergleiche ich die falsche Version @mattschaffrath ?!

mattschaffrath commented:
Du solltest dein (geforktes) Repository aktualisieren können, um den Pull Request zu aktualisieren. Du musst nur denselben Branch bearbeiten, den du für den Pull Request eingereicht hast.

Den Vergleich mache ich eigentlich immer über diesen "Files Changed"-Reiter oben im Pull Request und lasse mir das die Änderungen im Split-Screen nebeneinander anzeigen. Ich finde die von dir nicht gesehenen Änderungen aber gerade auch nicht und kann gerade auch nicht reproduzieren, wo ich die her habe 🤔 - keine Ahnung, was mir da passiert ist, aber keine Änderung, kein Beschwerden. 😉

**Zu dem Punkt 3) nochmal:**
* Ich selbst komme so gar nicht aus dem D&D-Lager, mich haben immer andere Systeme interessiert, weil die Regeln keiner Edition mich überzeugen konnten oder zu mir passten.
* In Deutschland gibt es nicht die starke Prägung der Rollenspielkultur durch D&D wie in den USA, bei der Übersetzung von englischen Spielen haben die meisten nach Gutdünken übersetzt.
* TBH selbst kloppt bewusst bekannte Begriffe in die Tonne, anstatt sie der Tradition wegen zu erhalten (e.g. round > minute).
* TBH ist ein komplett eigenständiges Regelwerk, das zwar Rollenspielerfahrung voraussetzt, aber zu nichts explizit 100% kompatibel sein will.
* In anderen Bereichen gibt es mit anderen Übersetzungen keine Probleme - z.B. verwendet(e) World of Warcraft im Englischen bekannte Vokabeln wie "Strenght" und "Constitution", die im Deutschen dann aber einfach zu "Stärke" und "Ausdauer" wurden, ohne dass da ein D&D-Geek aufgeschrien hätte.
* Mein persönliches Haupteinsatzgebiet für das System sind Abenteuer von Dungeon Crawl Classics, die sowieso so weit von D&D-Terminologie und -System entfernt sind, dass man um grobe Anpassungen gar nicht herum kommt.
* Meine Spieler sind meistens Rollenspieler auf Cons. Womit die "rechnen" hat vor allem damit zu tun, was die ansonsten meistens spielen - DSA, Shadowrun, Cthulhu, Pathfinder, Fate, Computerspiele aller Couleur und noch viel mehr...

Darum sehe ich *für mich* keinen Grund, beim Übersetzen die eingeführten Begriffe eines Spiels besonders zu beachten, das aus meiner Sicht nur eins unter vielen ist. Ich will das Ganze lieber als beliebigen englischen Text ohne Abhängigkeiten übersetzen. Und potentiell will ich einen Text haben, der auch ohne eine bestimmte Vorprägung auskommt.  
Wer das Spiel als Oldschool-D&D-Variante benutzen will oder zumindest von der Seite her geprägt ist, der hat bestimmt eine andere Sichtweise und andere Anforderungen.

*Strength* = *Stärke* finde ich eine eingängige Übersetzung, gerade auch weil es in vielen anderen Kontexten äquivalent bleibt (strength of will = Willensstärke, strong = stark, strength of a relationship = Stärke einer Beziehung etc.). "*Körperkraft*" finde ich dagegen bemühter und eingeschränkter und es kommt in meinem Wortschatz jenseits von DSA und Kreuzworträtseln eigentlich nicht vor. Vielleicht kommst du irgendwoher, wo man das jeden Tag sagt, meinem subjektiven Sprachgefühl entspricht es nicht.

"Constitution" würde ich auch spontan überall sonst mit "Ausdauer" oder "Körperlicher Verfassung" übersetzen, wenn nichts dagegen spricht. "Konstitution" ist für mich ein Wort, das Sportler, Politiker und arme Leute im Fernsehen, die ernst genommen werden wollen, verwenden. Bei einem Fantasy-Rollenspiel kommt es meinem Sprachgefühl fast wie ein "false friend" vor.

Wobei ich insgesamt auch ein wenig den Tick habe, mich eher von modern oder hochgestochen wirkenden Worten fernzuhalten und tendenziell älter oder weniger fachlich/fremdsprachlich klingende Begriffe auszuwählen (etwa, wie ich als mein Charakter oder auf einem LARP oder Mittelaltermarkt sprechen würde oder wie ich mir vorstelle, dass meine Großeltern aus der Weltkriegsgeneration geredet haben). Da steckt aber vielleicht auch einfach das Verlangen hinter, im Spiel die moderne Welt auch sprachlich etwas auszuklammern.

Andererseits bin ich kein Sprachexperte oder professioneller Übersetzer, also keine Ahnung, was ich mir da vielleicht auch einbilde oder vormache oder wie arg ich durch mein rheinisch-ripuarisches Sprachumfeld geblendet werde.

**in der Nähe**... tja.
Wie gesagt, bin ich mit dem Entfernungsbegriffen insgesamt nicht so zufrieden und hatte vorher über verschiedene Kandidaten nachgedacht, einschließlich, aber nicht beschränkt auf:
* **Close:** bei, neben, benachbart, im Nahkampf, in Nahkampfreichweite, auf Tuchfühlung, an jmd. dran, in Reichweite
* **Nearby:** nah, im Nahbereich, in der Nähe, in Nähe, in unmittelbarer Umgebung
* **Far-Away:** weiter weg, abseits, in Schussweite, auf Abstand, entfernt
* **Distant:** weit entfernt, im Fernbereich, in Sichtweite, fort

Was ich gern hätte:
* die Begriffe sollten eindeutig/nicht zu verwechseln/nicht zu ähnlich sein (wenn *close = nah dran*, *dann nearby != nah*)
* die Begriffe sollten möglichst knapp/prägnant sein
* die Begriffe sollten von der Grammatik her austauschbar sein (*ein benachbarter/naher/entfernter Gegner*, *ein Gegner in Reichweite/Wurfweite/Schussweite/Sichtweite*)
* die Begriffe sollten vielseitig zu verwenden sein und keine aufwändige Grammatik erfordern (vgl. *ein Ziel in Nähe/in Nähe des Ziels/bewegt sich in Nähe zum Ziel* vs. *ein Ziel nahebei/nahe beim Ziel/bewegt sich in den "nahebei"-Bereich des Ziels*)
* die Begriffe sollten in alltagssprachlichen Sätzen vorkommen können

Du siehst, dass ich mit meinem derzeitigen Stand die eigenen Anforderungen nur unzureichend erfülle... 😕

Wie immer: Ich versuche, darzustellen, wo ich mit meinem Standpunkt herkomme, bin dabei offen für Einwände, Argumente und Vorschläge.

ghostgroove commented:
**3):** An WoW habe ich jetzt gar nicht gedacht, aber da hast du natürlich total Recht. Ich würde sagen wir bleiben einfach bei deiner Version.

`the-black-hack.md:`
* **Bogen und Einhandwaffen in Ausrüstungsliste:** revidiert
* **Zeile 27: Kernmechanik** Komma wieder eingefügt
* **Zeile 543: Doppelgänger** C&P Fehler, korrigiert
* **Göttliche Anrufungen:** Anrufungen und Gebetbuch finde ich super! Man könnte noch wirken (von Wunder wirken) nutzen statt zaubern um auch hier ein Abgrenzung zu haben.
* **Zeile 417: Stille** Ich hatte mich am Original orientiert und das einzelne weg gelassen. Geht aber sicher beides.
* **Zeile 418: Sprechen/Reden* Die Toten habe ich tatsächlich übersehen. Ich hatte hier sprechen gewählt, da es dem Gespräch näher war.
* **Zeile 433: Gift neutralisieren** Habe ich mal umgekrempelt.
* **unauffindbare "Fehler":** Zeile 238, Zeile 416

**Entfernungen:** Das ist glaube ich das schwierigste Problem welches es zu lösen gilt. Weiter weg und weit entfernt finde ich hier eigentlich schon passend, in dem Array würde auch nah dran für Nearby gut passen, aber für Close fällt mir nichts ein. Bei dem benachbart, abseits und entfernt Array liegt das Problem dann bei Nearby, da nahebei nicht statt den anderen genutzt werden kann. Hm :thinking:

* **Close:** dicht, greifbar, Schlagweite
* **Nearby:** unweit, Hörweite
* **Far-Away:** weit, Sichtweite
* **Distant:** Horizont

Eine richtig gute Idee habe ich gerade auch nicht, aber vielleicht hast du jetzt ja eine :smiley:

ghostgroove commented:
@mattschaffrath neugierige Frage am Rande weil die Szene ja bekanntlich ein Dorf ist: Könnte es sein das ich die gerade in einem Podcast aus Paustenbach gehört habe? :thinking:

mattschaffrath commented:
Bin gerade noch einmal zum Drüberschauen gekommen, ich denke, das sollten wir jetzt erst einmal so abgeben. Bei den Entfernungen fehlt mir im Augenblick die große Eingebung, ich würde sagen "wir lassen das mal so", bis einen von uns die Muse küsst. Im Prinzip kann ja jeder von uns (oder jede und jeder sonst) irgendwann die Änderungen einpflegen, wenn der Genius ihn oder sie überkommt.

Danke auf jeden Fall für den Einsatz, den du bisher schon in das Projekt gesteckt hast. 👍 Und ja, ich habe mich heute auch in einem DORPCast aus Paustenbach (mit abenteuerlicher Tonqualität) gehört. Da hast du eine gute Stichprobe meines gern erwähnten "Sprachgefühls". 😉

@brunobord - thank you for your patience waiting for us prattling on in German.
 
We discussed some of the changes to be made. I, for my part, am satisfied with everything; so the latest status should be the final version for this pull request.

We also laid down some principles for "our" German version, so you can assume that potential future modifications by @ghostgroove have my blessings. 😇

brunobord commented:
> @brunobord - thank you for your patience waiting for us prattling on in German.

> We discussed some of the changes to be made. I, for my part, am satisfied with everything; so the latest status should be the final version for this pull request

I'm overly excited that you finally managed to handle this ; that's a fantastic moment to see contributors working together on these translations.

Anyway, I'd like to ask you a tiny little favor: editing the Changelog file and add a short entry describing the purpose of this pull-request and mentioning its number, that is to say, the number #62 ; as you can see on the current Changelog.

As soon as you give me the green light, I'll merge this and publish your latest version on the website.

ghostgroove commented:
@brunobord I added something to the Changelog and hope that's what you needed :smiley: also fixed a minor bug I missed in the last commit.

@mattschaffrath 
* **Zeile 238:** habe ich heute Morgen dann doch noch gefunden und korrigiert
* **Entfernungen:** Idee hierzu: Magst du mal die zwei Germanisten fragen? Für irgendwas muss das Studium doch gut sein :smile:

brunobord commented:
Your work is now online. Congratulations and thanks a million times for your efforts!
